### PR TITLE
Fix label lower case

### PR DESCRIPTION
**Pull request by conao3**
I made the label the lower case. See #3.
I do not know about `LoC` so please correct it after merging.

(Cargo.lock change is a message automatically inserted by cargo.)

### CODE REVIEW

File: src/main.rs [hunk 1]
```diff
@@ -27,12 +27,12 @@ const BILLION: i64 = 1_000_000_000;
...
 const THOUSAND: i64 = 1_000;
 
-const BLANKS: &'static str = "Blank lines";
+const BLANKS: &'static str = "blank lines";
 const BLUE: &'static str = "#007ec6";
-const CODE: &'static str = "LoC";
-const COMMENTS: &'static str = "Comments";
-const FILES: &'static str = "Files";
-const LINES: &'static str = "Total lines";
+const CODE: &'static str = "code lines";
```
Comment (XAMPPRocky) [inline src/main.rs]: I'd prefer if this was `lines of code`, I know it's different from the rest, but `code lines` doesn't sound correct to me.

**Comment by XAMPPRocky** [comment]
Thank you for your PR! I'm sorry that I didn't get to this sooner as there are now merge conflicts. If you could resolve them and address the feedback I've left. I'd be happy to merge this.

**Comment by conao3** [comment]
rebased!

File: src/main.rs (hunk 1 above)
Comment (conao3) [reply src/main.rs]: OK!

**Comment by conao3** [comment]
fix `code lines` to `lines of code`.
You also prefer to me use `lines of blanks` and `lines of comments`?

**Comment by XAMPPRocky** [comment]
No that's fine. Thank you again for your PR, and congrats on your first contribution! 🎉

**Comment by conao3** [comment]
OK. Thanks!

**Pull request merged**